CHANGELOG.md

```diff
@@ -12,8 +12,8 @@
 * Respect custom Realm paths when using `Realm.open` (#1392 and #1393).
 
 ### Internal
-* Upgrading to Realm Sync 2.0.0-rc28.
-* Upgrading to Realm Object Server 2.0.0-rc.4.
+* Upgrading to Realm Sync 2.0.2.
```

Comment:
I think we will do a 2.0.0-rc21 with this change. 2.0.0-rc20 was released yesterday.